src/libraries/Common/src/System/Security/Cryptography/MLDsaCng.Windows.cs

```diff
@@ -359,8 +359,13 @@ internal static MLDsaCng ImportPkcs8PrivateKey(byte[] source, out int bytesRead)
         /// <inheritdoc/>
         protected override void Dispose(bool disposing)
         {
-            _key.Dispose();
-            _key = null!;
+            if (disposing)
+            {
+                _key.Dispose();
+                _key = null!;
+            }
+
+            base.Dispose(disposing);
```

Comment:
I think MLDsaOpenSsl.Dispose calls base first.  Last does seem better, and since this PR is all about "do the same thing everywhere", seems like that one ought to be included.